frontend/packages/dev-console/integration-tests/features/addFlow/quick-search-add.feature

```diff
@@ -57,10 +57,11 @@ Feature: Provide quick search in Add page
              Then user is taken to the Topology page with "devfile-sample-git" workload created
 
 
-        @regression @to-do @odc-6467
+        @regression @odc-6467
         Scenario: Bindable resource in Quick Add: A-11-TC07
             Given user has installed Service Binding operator
-              And user has installed Crunchy Postgres for Kubernetes operator
+              And user is at developer perspective
+            Given user has installed Crunchy Postgres for Kubernetes operator
```

Comment:
just a nitpick, you now have 2 `Given`s in one chain

Reply:
Sorry, I missed it. Resolved @jrichter1